setup.py

```diff
@@ -30,7 +27,7 @@
         'matplotlib>=2.2.3',
         'morph-tool>=0.1.14',
         'morphio>=2.1.1',
-        'neurom>=2.0.0',
+        'neurom>=2.0.0.dev0',
```

Comment:
See pip docs: https://pip.readthedocs.io/en/stable/reference/pip_install/#pre-release-versions

Seems like a good fit here, but happy to discuss if this is not preferred.

Reply:
I see. Yes, that sounds nice. Could remove the `pip_pre = true` in `tox.ini` then ?

Reply:
Done